exposed-core/src/main/kotlin/org/jetbrains/exposed/sql/Query.kt

```diff
@@ -174,6 +212,9 @@ open class Query(override var set: FieldSet, where: Op<Boolean>?) : AbstractQuer
                 if (distinct) {
                     append("DISTINCT ")
                 }
+                distinctOn?.let { columns ->
+                    columns.appendTo(prefix = "DISTINCT ON (", postfix = ")") { +"${it.table.tableName}.${it.name}" }
```

Comment:
Let's add a space in the postfix to make the SQL query more readable?

```suggestion
                    columns.appendTo(prefix = "DISTINCT ON (", postfix = ") ") { +"${it.table.tableName}.${it.name}" }
```